test/browser_test.dart

```diff
@@ -7,7 +7,7 @@ import 'package:platform_detect/src/navigator.dart';
 import './constants.dart';
 
 void main() {
-  Browser browser;
+  Browser? browser;
```

Comment:
It looks like we always set `browser` at the top of every test. Would it make sense to make this a `late` variable so that we don't need to use the "bang operator" in all the lines below?

Reply:
I'll give it a try.

Reply:
Yep, it worked great. Thanks for pointing this out!